### PR TITLE
Minors fixes (french translation)

### PR DESCRIPTION
**Pull request by Alzam01**
A lot of small fixes

### CODE REVIEW

File: D&D_5e_Shaped/precompiled/components/quick_weapons/actions.html [hunk 1]
```diff
@@ -14,10 +14,10 @@
     <span class="sheet-de">Scha.</span>
     <span class="sheet-fr">Dmg</span>
   </div>
-  <div class="sheet-col-10 sheet-center sheet-small-label sheet-sub-header">
+  <div class="sheet-col-12 sheet-center sheet-small-label sheet-sub-header">
```
Comment (Alzam01) [inline D&D_5e_Shaped/precompiled/components/quick_weapons/actions.html:17]: "Reach" is not displaying properly, so I made it slightly bigger

File: D&D_5e_Shaped/precompiled/pages/spells.html [hunk 2]
```diff
@@ -32,7 +32,7 @@ <h4 class="sheet-center">
   <div class="sheet-shaped-row sheet-mar-b-sm">
     <div class="sheet-col-80 sheet-spell-table">
       <div class="sheet-shaped-row sheet-sub-header sheet-relative">
-        <div class="sheet-col-07 sheet-offset-15 sheet-vert-bottom sheet-center sheet-small-label sheet-pad-r-sm">
+        <div class="sheet-col-07 sheet-offset-18 sheet-vert-bottom sheet-center sheet-small-label sheet-pad-r-sm">
```
Comment (Alzam01) [inline D&D_5e_Shaped/precompiled/pages/spells.html:35]: You move all the lines in the dashboard to 18 but forgot this line

**Comment by Alzam01** [comment]
Is it possible to move "skills" a little bit to the left ?
skill
so the translation (compétences) can fit in ?
competences
Or do you want "skills" to stay where he is ?

**Comment by mlenser** [comment]
For the most part everything should stay where it is. Columns shouldn't adjust, positioning should adjust, but words should.

If I have to account for all languages then it would be impossible to have a compact layout. German for instance has crazy compound words.

Abbreviation would be my go to option. Maybe in some cases a larger column width will be ok, but I'd have to check it out myself.

File: D&D_5e_Shaped/precompiled/pages/inventory.html [hunk 3]
```diff
@@ -234,9 +234,9 @@ <h4 class="sheet-center">
   <div class="sheet-inventory-page-2">
     <div class="sheet-shaped-row sheet-sub-header">
       <div class="sheet-col-05 sheet-center sheet-small-label">
-        <span class="sheet-en">Row #</span>
-        <span class="sheet-de">Zeile #</span>
-        <span class="sheet-fr">Rangée #</span>
+        <span class="sheet-en">Row</span>
```
Comment (Alzam01) [inline D&D_5e_Shaped/precompiled/pages/inventory.html:237]: You delete "#" in the first inventory page, I assume you forget page 2 and 3

**Comment by Alzam01** [comment]
Well, I asked mainly because there is nothing in the way (on the left) and that I would like to not put a abbreviation here (aesthetic reason), but I understand that you don't want to move for every language (yeah, German is crazy sometimes). I'll look and try to find another word to put here, or I'll abbreviate it (I do have a lot of abbreviation already anyway)

Are the other fixes good, or do I need to change something ?

**Comment by Alzam01** [comment]
up

**Comment by mlenser** [comment]
Tremorsense is still too big - it can't line break. (shrunk the german one too, but it just goes into the next field).

Reach on actions can't expand. All the options have to add up to 100.

Armor shouldn't line break - I expanded the rows.

**Comment by Alzam01** [comment]
I'm gonna reduce Tremorsense a lit bit more then, no problem.

Make sense :) It's up to you then, it's for the english part anyway ^^

Btw, I have difficulty trying to switch from the french sheet to another language or from the spell page to another, is it related to the sheet, roll20 or do you have no idea what I'm speaking of ?

**Comment by mlenser** [comment]
I did temorsense.

Maybe you're seeing something other than I am on reach. It looks good to me in english. Maybe take a screenshot?

Um, Switching languages should be easy - just click the flag. There is somewhat of a slight delay, but that cannot be solved by me.

**Comment by Alzam01** [comment]
It's in NPC, core page :
reach

It's not only for switching languages : (this is a gif, click on it if it's not playing)
animation 0
I need to click two times on the same tab for changing page. I think it as something to do with the "tab" and "lang" attributes : I see two of them, and one is always "English" and "spells". Since I do have this problem on campaigns that use the 31 aug version of the sheet, and since I didn't had this problem last week, I assume that this is a bug with Roll20 ?

EDIT : Deleting the two "tab" attribute solve the problem (I tested on my campaign). I think I saw a report of that bug before somewhere... I didn't delete the attribute in your campaign, so you can try it yourself.

**Comment by Alzam01** [comment]
up, just in case you didn't see it

**Comment by mlenser** [comment]
The tab switching is a bug from roll20, not me. Find the attributes named "tab" in your attributes and abilities section and remove them all.

I'm leaving for vacation in a few hours, but I should be able to merge this in - though I'm just going to remove any resizing stuff for now as I don't have time to ensure it all works.

**Comment by mlenser** [comment]
Actually I won't have time, but I'll do it on Sunday so it'll be before the next release anyways.

**Comment by Alzam01** [comment]
Nice vacation then :)

**Comment by mlenser** [comment]
Oh, this is already merged. Ignore my earlier comments then.

Reach can't adjust, you'll notice some other language titles are off too. Titles should just be gotten rid of, really.
Let me know if there are any outstanding issues besides the tab issue (I've seen it many times, can't be fixed by me).

**Pull request merged**